src/test/regress/expected/start_stop_metadata_sync.out

```diff
@@ -270,6 +270,63 @@ SELECT count(*) > 0 FROM pg_class WHERE relname LIKE 'reference_table__' AND rel
  t
 (1 row)
 
+\c - - - :master_port
+-- test synchronization for hasmetadata flag
+SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
+ start_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT start_metadata_sync_to_node('localhost', :worker_2_port);
+ start_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+\c - - - :worker_1_port
+SELECT hasmetadata, metadatasynced FROM pg_dist_node WHERE nodeport IN (:worker_1_port, :worker_2_port) ORDER BY nodeport;
+ hasmetadata | metadatasynced
+---------------------------------------------------------------------
+ t           | t
+ t           | t
+(2 rows)
+
+\c - - - :worker_2_port
+SELECT hasmetadata, metadatasynced FROM pg_dist_node WHERE nodeport IN (:worker_1_port, :worker_2_port) ORDER BY nodeport;
+ hasmetadata | metadatasynced
+---------------------------------------------------------------------
+ t           | t
+ t           | t
+(2 rows)
+
+\c - - - :master_port
+SELECT stop_metadata_sync_to_node('localhost', :worker_1_port);
+NOTICE:  dropping metadata on the node (localhost,57637)
+ stop_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT stop_metadata_sync_to_node('localhost', :worker_2_port);
+NOTICE:  dropping metadata on the node (localhost,57638)
+ stop_metadata_sync_to_node
+---------------------------------------------------------------------
+
+(1 row)
+
+\c - - - :worker_1_port
+SELECT hasmetadata, metadatasynced FROM pg_dist_node WHERE nodeport IN (:worker_1_port, :worker_2_port) ORDER BY nodeport;
```

Comment:
can we do:
```
stop on worker 1
show worker 1
show worker 2
stop on worker 2
show worker 1
show worker 2
```

so that we can see on the first time we connect to worker2 we have applied the update